DEVREADME.md

```diff
@@ -2,6 +2,11 @@
 
 This file is intended for developers working on the internals of the framework. If you're just looking how to get started with the framework, see the [docs](./docs)
 
+## Installing dependencies
+
```

Comment:
Quite a few comments on this PR are actually about #684, making it difficult for me to judge why things were done and if they are correct.

Therefore I'd suggest first to fix and merge #684. After that I can pull in those changes from main and we can merge this one in.

Reply:
I already reviewed that PR, it looks good. This PR is missing a lot of changes from that PR. If you can pull in the latest changes from that PR I think that should also work? As discussed in #684 we're not going to merge that PR to main until 0.2.0 is released, to prevent a lot of unused dependencies being added.

We can also merge #684 into the `next` branch (just created it) and then we can point this PR also at the `next` branch. If that's desired @blu3beri can you point #684 to `next`? I don't seem to have permission to do that

Reply:
Merged those changed in from main in c6b919ec.